Mathlib/Analysis/NormedSpace/Basic.lean

```diff
@@ -341,22 +342,52 @@ protected theorem NormedSpace.unbounded_univ : ¬Bornology.IsBounded (univ : Set
   hx.not_le (hR x trivial)
 #align normed_space.unbounded_univ NormedSpace.unbounded_univ
 
-/-- A normed vector space over a nontrivially normed field is a noncompact space. This cannot be
-an instance because in order to apply it, Lean would have to search for `NormedSpace 𝕜 E` with
-unknown `𝕜`. We register this as an instance in two cases: `𝕜 = E` and `𝕜 = ℝ`. -/
-protected theorem NormedSpace.noncompactSpace : NoncompactSpace E :=
-  ⟨fun h => NormedSpace.unbounded_univ 𝕜 _ h.isBounded⟩
+protected lemma NormedSpace.cobounded_neBot : NeBot (cobounded E) := by
+  rw [neBot_iff, Ne.def, cobounded_eq_bot_iff, ← isBounded_univ]
+  exact NormedSpace.unbounded_univ 𝕜 E
+
+instance (priority := 100) NontriviallyNormedField.cobounded_neBot : NeBot (cobounded 𝕜) :=
+  NormedSpace.cobounded_neBot 𝕜 𝕜
+
+instance (priority := 100) RealNormedSpace.cobounded_neBot [NormedSpace ℝ E] :
+    NeBot (cobounded E) := NormedSpace.cobounded_neBot ℝ E
+
+instance (priority := 100) NontriviallyNormedField.infinite : Infinite 𝕜 :=
```

Comment:
Could you lower a little bit the priority of these, to 90 or 80? It looks like there are several more reasonable options to be tried before.

Reply:
Done.